tools/docker/env/Dockerfile

```diff
@@ -29,7 +34,9 @@ RUN dpkg --add-architecture i386 && \
 	# These are needed to build Linux kernel:
 	flex bison bc libelf-dev libssl-dev \
 	# qemu-user is required to run alien arch binaries in pkg/cover tests.
-	qemu-user \
+	qemu-user qemu-user-static \
+	# for building root image
+	debootstrap ssh-tools \
```

Comment:
Please move this to tools/docker/big-env/Dockerfile.
The idea behind syz-env is to keep it as minimal as possible to run presubmit smoke tests. We ask users to download it if they have any issues, and it's already quite large on my taste.

Reply:
done